pyroSAR/tests/test_drivers.py

```diff
@@ -193,7 +195,8 @@ def test_archive2(tmpdir, testdata):
         assert db.size == (1, 0)
         shp = os.path.join(str(tmpdir), 'db.shp')
         db.export2shp(shp)
-    
+
+    gc.collect(2)
```

Comment:
Interesting. I have never seen this before. Great you found a fix for this problem! I think it would be better to place this line somewhere directly in the `Archive` class so that users don't run into this problem and need to find the fix in this test. I think this could fit very well into the `Archive.close` method. What do you think?